highway_env/road/road.py

```diff
@@ -322,9 +323,11 @@ def straight_road_network(
             ]
             net.add_lane(
                 *nodes_str,
-                StraightLane(
+                lane = StraightLane(
                     origin, end, line_types=line_types, speed_limit=speed_limit
                 ),
+                weight = weight,
+                lane_type= LaneType.HIGHWAY
```

Comment:
I skal være sikre på at der ikke er andre der bruger denne funktion som så ikke er en highway

Reply:
Det har jeg tjekket for

Det er kun highway og exit som bruger den